plugins/lookup/k8s.py

```diff
@@ -125,13 +125,8 @@
 
     requirements:
       - "python >= 3.6"
-      - "openshift >= 0.6"
+      - "kubernetes >= 11.0.0"
```

Comment:
@gravesm Do you think we should bump this version to 12.0.0? Since this version contains a major change in the way of setting the config.

Reply:
Didn't we add some logic to handle this? That said, I'm not opposed to bumping our requirement to 12.0.0. I guess now seems like as good a time as any.